backend/app/services/message_forwarding_service.py

```
@@ -0,0 +1,290 @@
+                        subtask_ids=subtask_ids,
+                        content_snapshots=content_snapshots,
+                        note=request.note,
+                        priority=request.priority,
+                    )
+                    forwarded_count += 1
+
+                    # Record contact
+                    if recipient.type == "user":
+                        contact_service.record_contact(sender_user_id, recipient.id)
+
+                except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)